boa/contract.py

```diff
@@ -40,6 +39,14 @@
 from boa.vyper.decoder_utils import ByteAddressableStorage, decode_vyper_object
 
 
+try:
+    # `eth-stdlib` requires python 3.10 and above
+    from eth.codecs.abi import decode, encode
```

Comment:
can we import these with more qualified names? ex. `import decode as abi_decode, encode as abi_encode`